recipes-bsp/imx-seco/imx-seco.inc

```diff
@@ -0,0 +1,11 @@
+# Below SECO names are used by imx-boot and imx-seco recipes,
+# therefore they are extracted into a separate include file.
+#
+# NOTE: SECO_FIRMWARE_NAME defaults to UNSUPPORTED, which breaks
+# the build if the machine does not have a SECO support.
+SECO_FIRMWARE_NAME              ?= "UNSUPPORTED"
+SECO_FIRMWARE_NAME_mx8qm         = "mx8qmb0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8qxp        = "mx8qxb0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8qxpc0      = "mx8qxc0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8phantomdxl = "mx8qxb0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8dxl        = "mx8dxla0-ahab-container.img"
```

Comment:
Please move this to imx-base as this can be defined on the SoC basis and avoid this to be on the recipe.

Reply:
This header in now dropped, all defines are moved to `imx-base.inc`